### PR TITLE
changed helper text to control-label class

### PR DESCRIPTION
**Pull request by mozi47**
#### Summary
The interactive dialog field description is not styled properly in the dark theme. The color of the description field was not aligned with the dark theme.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/25936#issuecomment-1899342398
Jira https://mattermost.atlassian.net/browse/MM-56528

#### Screenshots

BEFORE:

Light Mode:
before1

Dark Mode:
before Dark1

AFTER:

Light Mode:

Dark Mode:

#### Release Note
There was a "help-text" class on the field description, and the class color was "$dark-gray". 
Removed the conflicting class (help-text) from the field description to resolve the black text color in the dark theme.

### CODE REVIEW

**Review by mickmister — COMMENTED** [review]
Thanks for this contribution @mozi47!

Looks like this change fixes the issue :+1:  I want to make sure this is the change we want to do first, as we may instead want to edit the `help-text` class definition instead of removing this. Thanks for investigating and implementing this fix!

File: webapp/channels/src/components/widgets/settings/setting.tsx
Comment (mozi47) [reply webapp/channels/src/components/widgets/settings/setting.tsx:43]: @asaadmahmood 
image

Comment (mickmister) [reply webapp/channels/src/components/widgets/settings/setting.tsx:43]: @asaadmahmood Do you know which css variable and at which opacity we should use here? Also I noticed this class is actually used in the GitHub plugin's modal too and is experiencing the same issue.

Comment (mickmister) [reply webapp/channels/src/components/widgets/settings/setting.tsx:43]: Hi @mozi47, can you please try this for the color to use?

```css
 color: rgba(var(--center-channel-color-rgb), 0.75);
```

Comment (asaadmahmood) [reply webapp/channels/src/components/widgets/settings/setting.tsx:43]: I think we should use the color mentioned above @mozi47

Comment (mozi47) [reply webapp/channels/src/components/widgets/settings/setting.tsx:43]: perfect. let me try that.

Comment (mozi47) [reply webapp/channels/src/components/widgets/settings/setting.tsx]: Updated:
image

image

**Review by mickmister — APPROVED** [review]
LGTM thanks @mozi47 🚀

**Comment by mozi47** [comment]
Just checking in on the status of my PR. It seems to have gone stale. Is there anything else needed to get it merged?

**Comment by mickmister** [comment]
@asaadmahmood Can you give this another look? Needs another approval before merge

**Review by asaadmahmood — APPROVED** [review]
LGTM

**Pull request merged**
